CONTRIBUTING.md

```diff
@@ -0,0 +1,331 @@
+# Contributing to Scipian
+
+Thank you for your interest in contributing to Scipian! There are many ways
+you can contribute and we appreciate all of them.
+
+* [Feature Requests](#feature-requests)
+* [Bug Reports](#bug-reports)
+* [Issue Triage](#issue-triage)
+* [Toolchain](#toolchain)
+* [Pull Requests](#pull-requests)
+* [Committing](#committing)
+* [Documentation](#documentation)
+
+## Feature Requests
+
+To request a change to the way that Scipian works, please open an issue inside 
+the [scipian/rfcs][rfcs] repo and we will be happy to discuss and see if it 
+meets the acceptance criteria for an initiative to be created.
+
+[rfcs]: https://github.com/scipian/rfcs
+
+## Bug Reports
+
+Let's face it, bugs are unfortunate, but they're a reality in software. We 
+cannot fix what we do not know about, so please report liberally. If you're not 
+sure something is a bug or not, feel free to file anyway.
+
+If you have a chance, please search existing issues, as it's possible someone 
+else has already reported your error. However, we won't mind if you 
+accidentally file a duplicate.
+
+We would like issues filed in the respective repo the bug belongs to, but if
+you're not sure which repo, default to the [scipian/community][community] repo.
+
+Please include the following in your issue:
+
+```
+<short summary of the bug>
+
+I tried this:
+
+<steps that caused the bug/error>
+
+I expected to see this happen:
+
+<explanation>
+
+Instead, this happened:
+
+<explanation>
+```
+
+These components are necessary in helping figure out what happened:
+
+1. What you did
+2. What you expected
+3. What happened instead.
```

Comment:
Either remove the period here or add it to the lines above. I think it should probably be removed though.
```suggestion
3. What happened instead
```